walkeeper/src/http/routes.rs

```diff
@@ -85,8 +83,8 @@ async fn timeline_status_handler(request: Request<Body>) -> Result<Response<Body
     };
 
     let status = TimelineStatus {
-        tenant_id,
-        timeline_id,
+        tenant_id: zttid.tenant_id,
```

Comment:
Can we store combined id directly?

Reply:
Not sure that's better, separate ids are still useful on their own.